src/routes/boot.ts

```diff
@@ -136,8 +138,11 @@ type BootMiddleware = (
 ) => Promise<Record<string, unknown>>;
 
 const geoSection = (req: FastifyRequest): BaseBoot['geo'] => {
+  const region = req.headers['x-client-region'] as string;
```

Comment:
Usually geo services provide continent code as well though one of the headers, did you check if one is available so we don't have to use/maintain above mapping?

Reply:
Yeah, Ido checked and mentioned that it is not part of the headers unfortunately. Yeah we can do either continent code or full name.

Reply:
Maybe code is better since `region` is a code as well?

Reply:
I agree, better consistency. Should be updated now 🚢